### PR TITLE
Tdr 3600 add permission to access s3 bucket

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: root_locals.tf [hunk 1]
```diff
@@ -100,6 +100,7 @@ locals {
   runtime_java_11    = "java11"
 
   upload_files_cloudfront_dirty_bucket_name = "${var.project}-upload-files-cloudfront-dirty-${local.environment}"
+  draft_metadata_bucket_name                = "${var.project}-draft-metadata-${local.environment}"
```
Comment (TomJKing) [inline root_locals.tf:103]: Is there a reason to define the bucket name here? Its only used within the `root_draft_metadata.tf` I would define it as a `local` within the `rootf_draft_metadata.tf` unless its needed else where in the code.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor comment.

Assume those permission work to allow the Lambda to read/write from the bucket.

File: root_locals.tf (hunk 1 above)
Comment (vimleshtna) [reply root_locals.tf]: Yeah, we can define it within the `rootf_draft_metadata.tf`

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**